src/diffusers/models/embeddings.py

```diff
@@ -152,29 +152,58 @@ def forward(self, latent):
 
 
 class TimestepEmbedding(nn.Module):
-    def __init__(self, in_channels: int, time_embed_dim: int, act_fn: str = "silu", out_dim: int = None):
+    def __init__(
+        self,
+        in_channels: int,
+        time_embed_dim: int,
+        act_fn: str = "silu",
+        out_dim: int = None,
+        act_2: bool = False,
```

Comment:
```suggestion
        act_2_fn: str: Optional[str] = None,
```
Can we make thisa string attribute for more future flexibility?